.ci/Jenkinsfile

```diff
@@ -295,7 +295,14 @@ def withCloudTestEnv(Closure body) {
   }
   // Masking
   withEnvMask(vars: maskedVars) {
-    body()
+    // Env variables to be populated as tag/labels for the cloud resources
+    // NOTE: values can contain only lowercase letters, numeric characters, underscores, and dashes
+    withEnv(["TF_VAR_CREATED_DATE_TIME=${getCurrentBuildTime().toLowerCase().replaceAll(':', '_')}",
+             "TF_VAR_CREATED_DATE=${new Date().format('yyyy-MM-dd')}",
```

Comment:
`TF_VAR_CREATED_DATE` and `TF_VAR_CREATED_DATE_TIME` seem to be redundant, can we stick to a single var?

Reply:
The main idea was to use only one but since Google does not allow special chars, the format `yyyy-MM-ddTHH:mm:ss` is not supported. As a consequence, I created:
* `TF_VAR_CREATED_DATE` to support `yyyy-MM-dd`.
* `TF_VAR_CREATED_DATE_TIME` to support Date/Time with some special format (replacing `:` for `_`), it's not ideal but if needed then it can be manipulated to transform the date to the original format `xs:datetime`.

I read about tags/labels and they suggest `Use too many tags rather than too few tags.` (see https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html#tag-best-practices)

Reply:
So far we support only two cloud providers, GCP and AWS, maybe we could stick to a form accepted by both of them? maybe Unix timestamp?